diesel/src/macros/macros_from_codegen.rs

```diff
@@ -2,7 +2,11 @@
 /// This macro can only be used in combination with the `diesel_codegen` or
 /// `diesel_codegen_syntex` crates. It will not work on its own.
 ///
-/// FIXME: Oh look we have a place to actually document this now.
+/// Queries the database for the names of all tables, and calls
```

Comment:
This sentence should be the first paragraph, so the rustdoc summary shows it instead of "This macro can only be used in combination …". Same for the doc comments below.